test/test_torch.py

```diff
@@ -7433,6 +7433,23 @@ def test_copy_broadcast(self, device) -> None:
         x.copy_(y)
         self.assertEqual(x[3], y)
 
+    def test_clamp(self, device):
+        x = torch.randn(100, 50)
+        y = torch.randn(100, 50)
+        z = torch.randn(100, 50)
```

Comment:
Let's compare against np.clip on a variety of sizes and both contiguous and noncontiguous tensors

make_tensor has a noncontiguous kwarg now to make this easy

Reply:
Also: if this supports broadcasting this should test that, too

Does it support scalar x tensor interactions?